TTHAnalysis/python/plotter/ttH-multilepton/systsUnc.txt

```diff
@@ -198,14 +222,28 @@ CMS_ttHl_Clos_m_bt_norm : $FRAll : .*3l_bl.* : none
-CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_ee.* : none
-CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_em.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.5\, kinMVA_2lss_ttV_withHj_rTT\,-1\,1)/1.24' 
-CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_mm.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.8\, kinMVA_2lss_ttV_withHj_rTT\,-1\,1)/1.35' 
+# CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss.*    : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(ttH_2lss_ifflav(LepGood1_pdgId\,LepGood2_pdgId\,1.5\,1.25\,1.0)\, kinMVA_2lss_ttbar_withBDTrTT\,-1\,1)/ttH_2lss_ifflav(LepGood1_pdgId\,LepGood2_pdgId\,1.2\,1.1\,1.0)' 
+# CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss_ee.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.5\, kinMVA_2lss_ttbar_withBDTrTT\,-1\,1)/1.2' 
+# CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss_em.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.25\, kinMVA_2lss_ttbar_withBDTrTT\,-1\,1)/1.1' 
+# CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss_mm.* : none
+# CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss.*    : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(ttH_2lss_ifflav(LepGood1_pdgId\,LepGood2_pdgId\,1.0\,1.5\,1.8)\, kinMVA_2lss_ttV_withHj_rTT\,-1\,1)/ttH_2lss_ifflav(LepGood1_pdgId\,LepGood2_pdgId\,1.0\,1.23\,1.35)' 
+# CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_ee.* : none
+# CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_em.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.5\, kinMVA_2lss_ttV_withHj_rTT\,-1\,1)/1.24' 
+# CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_mm.* : templateSymm ; Normalize=False, AddWeight='lnN1D_p1(1.8\, kinMVA_2lss_ttV_withHj_rTT\,-1\,1)/1.35' 
+
+CMS_ttHl16_Clos_e_shape : $FRAll  : .*2lss_ee.*_2016    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl16_Clos_m_shape : $FRAll  : .*2lss_mm.*_2016    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl16_Clos_e_shape : $FRAll  : .*2lss_em.*_2016    : templateSymm ; Normalize=False, AddWeight='1.15' 
+CMS_ttHl16_Clos_m_shape : $FRAll  : .*2lss_em.*_2016    : templateSymm ; Normalize=False, AddWeight='1.15' 
+CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss_ee.*_2017    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_mm.*_2017    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl17_Clos_e_shape : $FRAll  : .*2lss_em.*_2017    : templateSymm ; Normalize=False, AddWeight='1.15' 
+CMS_ttHl17_Clos_m_shape : $FRAll  : .*2lss_em.*_2017    : templateSymm ; Normalize=False, AddWeight='1.15' 
+CMS_ttHl18_Clos_e_shape : $FRAll  : .*2lss_ee.*_2018    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl18_Clos_m_shape : $FRAll  : .*2lss_mm.*_2018    : templateSymm ; Normalize=False, AddWeight='1.3' 
+CMS_ttHl18_Clos_e_shape : $FRAll  : .*2lss_em.*_2018    : templateSymm ; Normalize=False, AddWeight='1.15' 
+CMS_ttHl18_Clos_m_shape : $FRAll  : .*2lss_em.*_2018    : templateSymm ; Normalize=False, AddWeight='1.15' 
```

Comment:
placeholders?
the old _shape uncertainties were defined to be approximately normalized, while these are normalization only.

Reply:
Yes, these are placeholders until we have the final number... Will add a "FIXME" so we don't forget